File: app/src/androidTest/java/net/tlalka/android/fiszki/view/pages/LessonScorePage.java
package net.tlalka.android.fiszki.view.pages;

import net.tlalka.android.fiszki.R;

public class LessonScorePage extends AbstractPage {

    @Override
    public void valid() {
        isVisible(R.id.lesson_icon);
        isVisible(R.id.lesson_topic);
        isVisible(R.id.lesson_score_total);
        isVisible(R.id.lesson_score_incorrect);
        isVisible(R.id.lesson_score_repeat, R.string.lesson_score_repeat);
        isVisible(R.id.lesson_score_lessons, R.string.lesson_score_lessons);
    }

    public void clickRepeat() {
        clickItem(R.id.lesson_score_repeat, R.string.lesson_score_repeat);
    }

    public void clickLessons() {
        clickItem(R.id.lesson_score_lessons, R.string.lesson_score_lessons);
    }
}

File: app/src/androidTest/kotlin/net/tlalka/fiszki/view/pages/LessonScorePage.kt
package net.tlalka.fiszki.view.pages

import net.tlalka.fiszki.R

class LessonScorePage : AbstractPage() {

    override fun valid() {
        isVisible(R.id.lesson_icon)
        isVisible(R.id.lesson_topic)
        isVisible(R.id.lesson_score_total)
        isVisible(R.id.lesson_score_incorrect)
        isVisible(R.id.lesson_score_repeat, R.string.lesson_score_repeat)
        isVisible(R.id.lesson_score_lessons, R.string.lesson_score_lessons)
    }

    fun clickRepeat() {
        clickItem(R.id.lesson_score_repeat, R.string.lesson_score_repeat)
    }

    fun clickLessons() {
        clickItem(R.id.lesson_score_lessons, R.string.lesson_score_lessons)
    }
}
